lib/service/Service.ts

```diff
@@ -144,26 +144,35 @@ class Service extends EventEmitter {
   }
 
   /**
-   * Get a list of standing peer orders from the order book for a specified trading pair, or for all
-   * trading pairs if no pair is specified.
+   * Get a list of standing peer orders from the order book for a specified trading pair
    */
   public getOrders = ({ pairId, maxResults }: { pairId?: string, maxResults?: number }) => {
-    const ret: OrderArrays = {
-      buyOrders: [],
-      sellOrders: [],
+    checkArgument(pairId !== undefined && pairId !== '', 'pairId has to be specified');
```

Comment:
Likewise I think we can drop the `undefined` check here and just check for empty string. Also let's say `pairId must be specified` here just to match the language of the other `checkArgument` messages.